doc/Extensions/Applications.md

```diff
@@ -1651,6 +1652,24 @@ SNMP extend script to monitor your Redis Server
 extend redis /etc/snmp/redis.py
 ```
 
+# RRDCached
+
+SNMP extend script to monitor your RRDCached via snmp
```

Comment:
This documentation is incomplete.  Monitoring the rrdcached server that LibreNMS uses needs to be described.

Reply:
Ok, you mean adding in this part, some information regarding local rrdcached socket, right? Ok, I'll add it in.

I think we should add some info here also, right?
https://docs.librenms.org/Extensions/RRDCached-Security/

Regarding security, it is preferrable to use Snmp rather than opening up 42217 directly... right, do you agree?

Do you want me to add in some text there too?

Reply:
Honestly, if you are monitoring the rrdcached server LibreNMS is using, it does not make any sense to use SNMP to me.
You are already connecting to 42217 for the polling process.  For security, I would just lock it down to the IPs of the pollers via firewall rules.  I am not sure the point of using nginx to proxy, you cannot encrypt the stream or add authentication, so it is basically acting as a firewall and maybe adding overhead.  Perhaps you gain some logging, but I'm unsure.

Reply:
Hi, @murrant, I agree with you for localhost.

But this CHG is for polling a remote rrdcached service on another server, where we dont have access to rrdcached thru port 42217 for any reason.

I think that, librenms shouldn't just fallback to local unix socket, especially for cases of remote servers service polling, we should add Snmp query to remote 'extend' rrdcached service.

This was my case, I have librenms on docker, and I have rrdcache on another server that I want to monitor, but I didn't want to open the port for security reasons, what I did was, extend snmp in that server, but without this code, I was simply, monitoring local rrdcache, It took me a while to figure this out, and understand that the rrdcached graphs for the other server, were equal to LibreNMS rrdcached 'localhost' graphs.

The atual code, forces us to query rrdcached of localhost, so the information is not from the remote, and may lead to confusion and it's a wrong information...